modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/UpgradeCatchAllCheck.java

```diff
@@ -396,6 +399,27 @@ private String _formatParameters(
 			return newContent;
 		}
 
+		if (to.isEmpty()) {
+			String newJavaMethodContent = StringUtil.replaceFirst(
+				javaMethodContent, methodCall, "");
```

Comment:
@nicolas-sss, why not using `StringPool.BLANK` instead `""` to keep consistency with existing code?

Reply:
Resolved